datadog/resource_datadog_integration_pagerduty_service_object.go

```diff
@@ -0,0 +1,100 @@
+package datadog
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/zorkian/go-datadog-api"
+)
+
+const maskedSecret = "*****"
+
+func resourceDatadogIntegrationPagerdutySO() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogIntegrationPagerdutySOCreate,
+		Read:   resourceDatadogIntegrationPagerdutySORead,
+		Update: resourceDatadogIntegrationPagerdutySOUpdate,
+		Delete: resourceDatadogIntegrationPagerdutySODelete,
+		// TODO: importer
```

Comment:
?

Reply:
Ah, yes, that's my WIP TODO that I will solve and remove... basically I need to make sure that `terraform import` works well with the service object resource.

Reply:
Due to the fact that the Datadog API doesn't return service keys, it's not really meaningful to import this resource, so I explicitly disabled it by setting `Importer: nil`. I'm marking this discussion as resolved.